packages/ckeditor5-ui/src/editorui/poweredby.ts

```diff
@@ -94,6 +95,12 @@ export default class PoweredBy extends DomEmitterMixin() {
 	private _handleEditorReady(): void {
 		const editor = this.editor;
 
+		const licenseKey = this.editor.config.get( 'licenseKey' );
```

Comment:
```suggestion
		const licenseKey = editor.config.get( 'licenseKey' );
```

It's preassigned just a line above :D